netcdf-java-platform/build.gradle

```diff
@@ -11,7 +11,6 @@ javaPlatform {
 dependencies {
   def awsVersion = '2.17.290'
   api enforcedPlatform("software.amazon.awssdk:bom:${awsVersion}")
-  api enforcedPlatform('com.fasterxml.jackson:jackson-bom:2.14.0-rc1')
```

Comment:
I would keep the `jackson-bom`, as it can control the transitive dependency versions. In order to have `netcdfAll` find the dependencies, you'll need to keep the the `jackson-core` and `jackson-databind` api lines below, and define a `jacksonVersion` variable like I did with `awsVersion` for those two lines to reference. Something like:

```
  def awsVersion = '2.17.290'
  def jacksonVersion = '2.14.0-rc1'
  api enforcedPlatform("software.amazon.awssdk:bom:${awsVersion}")
  api enforcedPlatform("com.fasterxml.jackson:jackson-bom:${jacksonVersion}")
  constraints {
    ...
    // netcdfAll (everything else is happy with the jackson BOM)
    api "com.fasterxml.jackson.core:jackson-core:${jacksonVersion}"
    api "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
    ...
```

Reply:
We're not actually even putting zarr in netcdfAll but #1045 :(

Reply:
also do you know _why_ it doesn't work for netcdfAll?

Reply:
I think it does not work because we try to get away with not having essentially empty projects for netcdfAll, toolsUI, and ncIdv (a folder and single build.gradle for each one). We create custom configurations for those projects and attach dependencies to those configurations, and that does not seem to be able to work with dependencies that are BOMs within platform projects. I bet if you created actual subprojects and declared dependencies using implementation or runtimeOnly, it would work as expected (but then you'd have more directories cluttering up the layout).